ee/dtr/architecture.md

```diff
@@ -11,76 +11,62 @@ Docker Universal Control Plane cluster.
-are created:
+## DTR volumes
 
-| Name   | Type    | Description                                                                            |
-|:-------|:--------|:---------------------------------------------------------------------------------------|
-| dtr-ol | overlay | Allows DTR components running on different nodes to communicate, to replicate DTR data |
+DTR uses a selection of volumes for persisting data.
 
-
-## Volumes used by DTR
-
-DTR uses these named volumes for persisting data:
-
-| Volume name                         | Description                                                                      |
+| Volume Name                         | Description                                                                      |
 |:------------------------------------|:---------------------------------------------------------------------------------|
 | dtr-ca-&lt;replica_id&gt;           | Root key material for the DTR root CA that issues certificates                   |
 | dtr-notary-&lt;replica_id&gt;       | Certificate and keys for the Notary components                                   |
 | dtr-postgres-&lt;replica_id&gt;     | Vulnerability scans data                                                         |
-| dtr-registry-&lt;replica_id&gt;     | Docker images data, if DTR is configured to store images on the local filesystem |
+| dtr-registry-&lt;replica_id&gt;     | Docker images data (if DTR is configured to store images on the local filesystem) |
 | dtr-rethink-&lt;replica_id&gt;      | Repository metadata                                                              |
-| dtr-nfs-registry-&lt;replica_id&gt; | Docker images data, if DTR is configured to store images on NFS                  |
+| dtr-nfs-registry-&lt;replica_id&gt; | Docker images data (if DTR is configured to store images on NFS)                  |
 
-You can customize the volume driver used for these volumes, by creating the
-volumes before installing DTR. During the installation, DTR checks which volumes
-don't exist in the node, and creates them using the default volume driver.
+To customize the volume driver for these volumes, create the volumes prior to installing DTR. Then, during the installation, DTR will determine which existing volumes do not yet exist on the node and create them there using the default volume driver.
 
 By default, the data for these volumes can be found at
 `/var/lib/docker/volumes/<volume-name>/_data`.
 
 ## Image storage
 
-By default, Docker Trusted Registry stores images on the filesystem of the node
-where it is running, but you should configure it to use a centralized storage
-backend.
+By default, Docker Trusted Registry stores images on the filesystem of the node on which it is running. It should, though, be configured to use a centralized storage backend.
 
 ![](images/architecture-3.svg)
 
-DTR supports these storage backends:
+DTR supports the following storage backends:
```

Comment:
This list is outdated. It should be:
* NFS
* Amazon S3
* Microsoft Azure
* OpenStack Swift
* Google Cloud Storage